django/db/backends/postgresql_psycopg2/base.py

```diff
@@ -231,6 +235,23 @@ def is_usable(self):
             return True
 
     @cached_property
+    def _nodb_connection(self):
+        nodb_connection = super(DatabaseWrapper, self)._nodb_connection
+        try:
+            nodb_connection.ensure_connection()
+        except (DatabaseError, WrappedDatabaseError):
+            warnings.warn(
+                "Unable to create a connection to the 'postgres' database, using "
```

Comment:
maybe something like "Normally Django will use a connection to the 'postgres' database to avoid running initialization queries against the production database when it's not needed (for example, when running tests). Django was unable to create a connection to the 'postgres' database and will use the default database instead."